app/templates/base_error.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>{{ title if title else "Error"}}</title>

```

Comment:
nice … but why not just hardcode it? it's not like you're using it (unless i missed it)